### PR TITLE
some style nits

### PR DESCRIPTION
**Pull request by ryan-williams**
basically just cleared up the intellij warnings in two files I was working in for #1277:
- unused imports
- many no-op `else records` branches of RDD-instrumenting-`if`-statements 
- some `.size`->`.length` changes
- some unnecessary `.toSeq`s

### CODE REVIEW

File: adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala [hunk 1]
```diff
@@ -480,7 +472,10 @@ class ADAMContext private (@transient val sc: SparkContext) extends Serializable
       sc.newAPIHadoopFile(filePath, classOf[AnySAMInputFormat], classOf[LongWritable],
         classOf[SAMRecordWritable], ContextUtil.getConfiguration(job))
     }
-    if (Metrics.isRecording) records.instrument() else records
+
+    if (Metrics.isRecording)
```
Comment (heuermh) [inline adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala]: Though I'll probably get downvoted, I prefer to always have curly braces
```scala
if (Metrics.isRecording) {
  records.instrument()
}
```

Comment (fnothaft) [inline adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala]: I'm also an "always braces" type.

**Comment by heuermh** [comment]
Would it be ok to close this without merging?

**Comment by fnothaft** [comment]
@heueh +1

**Pull request closed without merging**